misk-hibernate-testing/src/main/kotlin/misk/hibernate/HibernateTestingModule.kt

```diff
@@ -21,49 +23,53 @@ import kotlin.reflect.KClass
  */
 class HibernateTestingModule(
   private val qualifier: KClass<out Annotation>,
+  private val config: DataSourceConfig,
```

Comment:
Looks good, but this will break services on the next update. Do we want to update everything, or make this optional?

Reply:
Yep, good call out. I was planning to do a big batch fix once merged.